enhancements/baremetal-IPI/baremetal.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Neat-Enhancement-Idea
+authors:
+  - "@sadasu"
+reviewers:
+  - TBD
+  - "@abhinavdahiya"
+approvers:
+  - TBD
+  - "@abhinavdahiya"
+creation-date: 2019-11-06
+last-updated: 2019-11-06
+status: implemented
+---
+
+# Adding Baremetal IPI capabilities to OpenShift
+
+This enhancement serves to provide the context for a whole slew of features
+and enhancements that will follow to make Baremetal IPI deployments via
+OpenShift a reality.
+
+At the time of this writing, code for some of the enhancements have already
+merged, some are in progress and others are yet to me implemented. References
+to all these features in different stages of development will be provided
+below.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding
```

Comment:
Is this boilerplate?

Reply:
Yes. Kept it there for now as guidance if reviewers feel I should be adding something to a section.